packages/utils/regl-renderer/types/geometry-utils-V1/cagToGeometries.d.ts

```diff
@@ -0,0 +1,8 @@
+export = cagToGeometries;
```

Comment:
Please remove as V1 is not supported / tested. These functions will be removed in the future.

Reply:
Removed.